Frontify/guideline-blocks
# fix(Checklist): stop contenteditable warning

Clickup ticket: https://app.clickup.com/t/2xd5j6y

Suppress content editable warning and refactor decorateLabelChildren to prevent key warning on empty input.

To test:
- Add the block with at least one item. If no warning appears then contentEditable issue is resolved.
- Add a checklist item, remove the text, check the box. If no warning appears key issue is resolved (Note that there is still a single key warning after this change, but this comes from the OrderableList component)
